imports/api/collections/jobs.js

```diff
@@ -60,7 +60,12 @@ Job = new SimpleSchema({
     autoValue: function(){
       return new Date().getTime();
     }
-  }
+  },
+  applications: {
```

Comment:
I added this fields to the schema, all students that apply to this job will have their id in here